component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/record/AvroSchemaBuilder.java

```diff
@@ -73,42 +74,44 @@ public class AvroSchemaBuilder implements Schema.Builder {
             new AvroSchema(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.BOOLEAN));
 
     private static final AvroSchema BYTES_SCHEMA_NULLABLE = new AvroSchema(org.apache.avro.Schema
-            .createUnion(asList(NULL_SCHEMA, org.apache.avro.Schema.create(org.apache.avro.Schema.Type.BYTES))));
+            .createUnion(asList(org.apache.avro.Schema.create(org.apache.avro.Schema.Type.BYTES), NULL_SCHEMA)));
```

Comment:
about the reason, the comment in https://jira.talendforge.org/browse/TCOMP-2094 explained it:
```
DefaultValue breaking change
Add to fix breaking change since 1.9.0 in AvroRecordBuilderTest on field defaultValue validation. Before 1.9.0, validation for defaultValue was disabled and became enabled in 1.9.0.
The validation consists to check default value provided against the field's type. But when a field is nullable, the schema is smtg like union["null", "type"] and the validation process checks only first type provided.
So, if we've have a nullable field with a defaultValue other than null the validation will fail.
```

Reply:
about this questions:
```
Did you tested the changes against the connectors' repositories? (no pb w/ processingfunc?)
```
tck connectors are fixed in https://jira.talendforge.org/browse/TDI-50138, have tested:
1. have passed all junits test for connectors-se/ee/cloud-components.
2. studio integrtion test : passed QA test : https://jira.talendforge.org/browse/TDI-50245
3. cloud integration test : not passed : https://jira.talendforge.org/browse/TDI-50246